pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1306,6 +1306,8 @@ type BRConfig struct {
 	StatusAddr string `json:"statusAddr,omitempty"`
 	// Concurrency is the size of thread pool on each node that execute the backup task
 	Concurrency *uint32 `json:"concurrency,omitempty"`
+	// ChecksumConcurrency is the size of thread pool on each node that execute the checksum task
+	ChecksumConcurrency *uint32 `json:"checksum-concurrency,omitempty"`
```

Comment:
```suggestion
	ChecksumConcurrency *uint32 `json:"checksumConcurrency,omitempty"`
```

Reply:
It seems `.spec.br.options` can meet my requirement. this pr will be closed.